web/pages/[workspaceSlug]/projects/[projectId]/settings/states.tsx

```diff
@@ -5,17 +5,29 @@ import { ProjectSettingLayout } from "layouts/settings-layout";
 // components
 import { ProjectSettingStateList } from "components/states";
 import { ProjectSettingHeader } from "components/headers";
+import { PageHead } from "components/core";
 // types
 import { NextPageWithLayout } from "lib/types";
+// hook
+import { useProject } from "hooks/store";
 
-const StatesSettingsPage: NextPageWithLayout = () => (
-  <div className="w-full gap-10 overflow-y-auto py-8 pr-9">
-    <div className="flex items-center border-b border-custom-border-100 py-3.5">
-      <h3 className="text-xl font-medium">States</h3>
-    </div>
-    <ProjectSettingStateList />
-  </div>
-);
+const StatesSettingsPage: NextPageWithLayout = () => {
+  // store
+  const { currentProjectDetails } = useProject();
+  // derived values
+  const pageTitle = currentProjectDetails?.name ? `${currentProjectDetails?.name} - States` : undefined;
+  return (
+    <>
+      <PageHead title={pageTitle} />
```

Comment:
can we not render page head if currentProjectDetails is undefined

Reply:
If "currentProjectDetails" is undefined, it will render the default page head.